### DISCUSSION TITLE
3주차 미팅

### DISCUSSION THREAD

**Discussion by sejoonkimmm** [GunGonGamLee/ft_transcendence#62] [General]
# 해야할 것
---
- 42OAuth, Google login 구현
- Figma 디자인 css로 뽑아내기 / tailwind로 뽑아내기

---

# 의논할 것
- 코어타임, 역할 분담
- 쌩 css? tailwind?

**Comment by nyj001012** [comment]
## 3주차 할 일
- 전적 페이지 - 사용자 게임  1 vs 1 모드 개요 페이지 css 구현

**Comment by bluedog129** [comment]
# 3주차 할 일

* 로그인페이지 - css구현, js구현
